exercise/templates/exercise/staff/_assessment_panel.html

```diff
@@ -84,38 +84,41 @@
 			{# Body: left column (submitted files) #}
 			{% if has_files %}
 			<div class="col-sm-6">
-				<div class="form-group">
-					<select id="submitted-file-select" class="form-control" autocomplete="off">
-						{% for file in submission.files.all %}
-						<option value="{{ file.id }}" {% if forloop.first %}selected{% endif %}>{{ file.filename }}</option>
-						{% endfor %}
-					</select>
-				</div>
+				<ul class="nav nav-tabs" role="tablist">
+					{% for file in submission.files.all %}
+					<li class="nav-item {% if forloop.first %}active{% endif %}" role="presentation">
+						<a class="nav-link" href="#{{ file.id }}" role="tab" data-toggle="tab">{{ file.filename }}</a>
```

Comment:
The id of the file tab could be `file-{{ file.id }}` so that the id number can not collide with other objects in case the page lists other objects too. For example, file.id could be the same number as submission.id. HTML ids must be unique in the document.

`<a>` is missing `aria-controls="file-{{ file.id }}"`

```suggestion
						<a class="nav-link" href="#file-{{ file.id }}" role="tab" aria-controls="file-{{ file.id }}" data-toggle="tab">{{ file.filename }}</a>
```

https://getbootstrap.com/docs/3.4/javascript/#tabs